config/deploy.rb

```diff
@@ -74,3 +80,4 @@
 end
 
 before 'bundler:install', 'jruby_bundle_install'
+after 'bundler:install', 'install_foreman'
```

Comment:
Could this be `before 'deploy:update_systemd_scripts', 'install_foreman'`?  We don't need it unless we're updating the systemd config.

Reply:
I think we always want it, in case someone wants to do it manually?